2-mandatory/4-people-i-know.js

```diff
@@ -378,13 +378,24 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+let friendsFilter = friends.filter(function(e){
+  if(e.age >= 35){
+    thirtyFiveOrOlder.push(e)
+  }
+})
 
```

Comment:
would be great if parameter name could be more descriptive

Reply:
Agreed with Iwona here!